lib/plugins/chat.js

```diff
@@ -182,8 +182,12 @@ function inject (bot, options) {
       lookedAtBlock: position
     })
 
-    const [packet] = await once(bot._client, 'tab_complete')
-    return packet.matches
+    try {
+      const [packet] = await onceWithCleanup(bot._client, 'tab_complete', { timeout })
+      return packet.matches
+    } catch {
+      return []
```

Comment:
Yes I think let's not try catch here. The user should handle the error appropriately (for example maybe by retrying)

Reply:
Done